tests/VObject/Recur/RRuleIteratorTest.php

```diff
@@ -9,6 +9,18 @@
 
 class RRuleIteratorTest extends TestCase
 {
+    /**
+     * @expectedException \Sabre\VObject\InvalidDataException
```

Comment:
Could you use instead

```
$this->expectException(\Sabre\VObject\InvalidDataException::class);
```

The annotation has been removed in PHPUnit 8 (and deprecated in PHPUnit 7). Sabre/vobject has already begin to move to PHPUnit 8, it will be an issue later

Reply:
You're right, just updated